src/fictionalpancake/turbospork/paint/GraphicsHandler.java

```diff
@@ -0,0 +1,223 @@
+package fictionalpancake.turbospork.paint;
+
+import fictionalpancake.turbospork.*;
+
+import java.util.List;
+import java.util.Random;
+
+public class GraphicsHandler {
+
+    private GameHandler gameHandler;
+    private Node selectedNode;
+    private Node lastSelected;
+    private Node lastAttacked;
+    private long lastPaint;
+    private int xOffset;
+    private int yOffset;
+    private double scale;
+
+    public GraphicsHandler(GameHandler gameHandler) {
+        this.gameHandler = gameHandler;
+    }
+
+    public void paint(IPainter g) {
+        checkScaling(g);
+        java.util.List<Node> nodes = gameHandler.getNodes();
+        java.util.List<UnitGroup> groups = gameHandler.getUnitGroups();
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'java.util.List' due to existing import 'java.util.List'](https://www.codacy.com/app/vpzomtrrfrt/turbo-spork/file/2611259179/issues/source?bid=3248828&fileBranchId=3310821#l26)